### DISCUSSION TITLE
Rename repo to `open62541-rs`

### DISCUSSION THREAD

**Discussion by uklotzde** [HMIProject/open62541#174] [General]
Context:

- https://github.com/HMIProject/open62541/discussions/172
- https://github.com/basyskom-dege/open62541_rust

I just noticed that renaming might be necessary when forking this repo. Keeping a fork of the original C library is not an uncommon situation for contributors.

We should have named the repo `open62541-rs` to prevent such naming conflicts. By forking https://github.com/open62541/open62541 ourselves we might have noticed this earlier. We could still rename it.

The name of the crate is fine.

**Comment by sgoll** [comment]
When forking on GitHub, you can pick a custom repository name for your clone of any repository. I am not sure that this is such a relevant problem. I have run across this myself and saw no issue in picking [open62541-rs](https://github.com/sgoll/open62541-rs) for my copy of the code.
